core/services/synchronization/telem/telem_head_report.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink/v2/core/services/synchronization/telem";
+
+package telem;
+
+message HeadReportRequest  {
+  uint64 chain_id = 1;
```

Comment:
looks like this refers to EVM chain IDs

it feels like we should either add more identifiers to accommodate non-evm chains, or explicitly call out this is just for EVM (i.e. rename `HeadReportRequest` to `EVMHeadReportRequest`, `HeadReporter` to `EVMHeadReporter` etc)

Reply:
Hmm, good call. Do we have head trackers for the other chains? I see the interface returning `evmtypes.Head`, but I'm all for making this as generic as possible. Maybe making this field be `string chain = 1`, and to contain chain_ids for evm, and whatever may identify others?

Reply:
Actually, I think we can drop this field completely; the endpoints generated with `monitoringEndpointGen.GenMonitoringEndpoint` are already chain-specific, and get this from metadata

Reply:
good observation, that sounds reasonable to me 👍

Reply:
We decided to keep this field to track chain in case of any NOP misconfiguration
I also made this field string chain = 1